drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -237,12 +231,12 @@ static int rtc_stm32_stop(const struct device *dev)
 }
 
 #if !defined(COUNTER_NO_DATE)
-tick_t rtc_stm32_read(const struct device *dev)
+counter_tick_t rtc_stm32_read(const struct device *dev)
```

Comment:
Replace `counter_tick_t` with `counter_ticks_t` in this whole source file. Error reported by CI build tests.